docs/known_bugs.rst

```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Spectrum does not appear when running on a VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running Jdaviz on a virtual Linux machine, the spectrum may not appear
+in the spectrum viewer. This is a known bug in an underlying package. Until
```

Comment:
Can we link to that issue upstream?

Reply:
We could, I just didn't think the average user would care.